lib/actions.js

```diff
@@ -2,10 +2,54 @@
 
 var Promise = require('bluebird');
 var namespaceTransform = require('../util/namespaceTransform');
-var Dispatcher;
+var _ = require('lodash');
 
 function noop() {}
 
+function BaseActions(namespace, fluxApp) {
+  this.namespace = namespace;
+  this._fluxApp = fluxApp || require('./');
```

Comment:
If this is required for our code, I would rather refactor than require a file inline.